sale_order_pricelist_commitment_date/__manifest__.py

```diff
@@ -10,5 +10,5 @@
     "license": "LGPL-3",
     "installable": True,
     "depends": ["sale_order_secondary_unit"],
-    "data": ["views/sale_view.xml"],
+    "data": ["views/sale_order_view.xml"],
```

Comment:
```suggestion
    "data": ["views/sale_order_views.xml"],
```

Following the OCA convention: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming